migrations/educandu-2022-02-11-01-remove-empty-cdn-resources.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable camelcase, no-await-in-loop, no-console */
+export default class Educandu_2022_02_11_01_remove_empty_cdn_resources {
+  constructor(db) {
+    this.db = db;
+  }
+
+  async updateDocumentsInCollection(collectionName) {
+    const projections = await this.db.collection(collectionName).aggregate([{ $project: { cdnResources: 1 } }]).toArray();
```

Comment:
As there are very few cases, maybe this would be more efficient: find: `{ cdnResources: { $elemMatch: { $eq: '' } } }`

Reply:
Updated and included lessons, thanks!